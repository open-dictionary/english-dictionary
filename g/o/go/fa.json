{
	"word": "go",
	"definitions": [
		{
			"definition": "خرج شدن"
		},
		{
			"definition": "نوعى پاس با دويدن دريافت کننده به جلو( فوتبال امريکايى)"
		},
		{
			"definition": "سفت"
		},
		{
			"definition": "روانه ساختن"
		},
		{
			"definition": "رهسپار شدن"
		},
		{
			"definition": "عزيمت کردن"
		},
		{
			"definition": "گذشتن"
		},
		{
			"definition": "عبور کردن"
		},
		{
			"definition": "کارکردن"
		},
		{
			"definition": "گشتن"
		},
		{
			"definition": "رواج داشتن"
		},
		{
			"definition": "تمام شدن"
		},
		{
			"definition": "راه رفتن"
		},
		{
			"definition": "نابود شدن"
		},
		{
			"definition": "روى دادن"
		},
		{
			"definition": "بران بودن"
		},
		{
			"definition": "درصدد بودن"
		},
		{
			"definition": "راهى شدن"
		}
	]
}