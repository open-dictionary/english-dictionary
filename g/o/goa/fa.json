{
	"word": "goa",
	"definitions": [
		{
			"definition": "اهوى تبت"
		}
	]
}