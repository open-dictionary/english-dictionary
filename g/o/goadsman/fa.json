{
	"word": "goadsman",
	"definitions": [
		{
			"definition": "گاوچران"
		},
		{
			"definition": "سيخک زن"
		}
	]
}