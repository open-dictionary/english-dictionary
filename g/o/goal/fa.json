{
	"word": "goal",
	"definitions": [
		{
			"definition": "(در فوتبال )دروازه"
		},
		{
			"definition": "دروازه بان"
		},
		{
			"definition": "مقصد"
		},
		{
			"definition": "هدف"
		},
		{
			"definition": "گل زدن"
		},
		{
			"definition": "هدفى در پيش داشتن"
		}
	]
}