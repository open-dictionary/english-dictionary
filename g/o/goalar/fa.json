{
	"word": "goalar",
	"definitions": [
		{
			"definition": "دروازه بان"
		}
	]
}