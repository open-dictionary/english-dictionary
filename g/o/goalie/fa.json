{
	"word": "goalie",
	"definitions": [
		{
			"definition": "دروازه بان"
		}
	]
}