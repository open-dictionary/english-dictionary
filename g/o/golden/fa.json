{
	"word": "golden",
	"definitions": [
		{
			"definition": "طلائى"
		},
		{
			"definition": "طلايى"
		},
		{
			"definition": "زرين"
		},
		{
			"definition": "اعلا"
		},
		{
			"definition": "درخشنده"
		}
	]
}